wled00/mqtt.cpp

```diff
@@ -195,4 +270,145 @@ bool initMqtt()
   mqtt->connect();
   return true;
 }
+
+void otaUpdate(String url)
+{
+  DEBUG_PRINT(F("OTA update from URL: "));
+  DEBUG_PRINTLN(url);
+
+  // make client for HTTP request
+  HTTPClient http;
```

Comment:
Can this also handle HTTPS?

Reply:
I haven't tested that out, it might be worth trying